packages/vscode/src/index.ts

```diff
@@ -35,6 +35,10 @@ export async function activate(ext: ExtensionContext) {
         files: 'nuxt.config',
         fields: 'unocss',
       }),
+      sourceObjectFields({
+        files: 'aleph.config',
+        fields: 'atomicCSS',
```

Comment:
i am considering to rename `atomicCSS ` to `unocss`

Reply:
Yeah I think `unocss` is better for explictness 👍